### PR TITLE
Added the new bio reactor model

### PR DESCRIPTION
**Pull request by ccgould**
All animations ,  sounds and textures have been added to the game

### CODE REVIEW

File: CyclopsBioReactor/CyclopsBioReactor.csproj [hunk 1]
```diff
@@ -39,10 +39,10 @@
       <HintPath>..\DependenciesUnity5\0Harmony-1.2.0.1.dll</HintPath>
     </Reference>
     <Reference Include="Assembly-CSharp-firstpass_publicized">
-      <HintPath>..\DependenciesUnity5\Assembly-CSharp-firstpass_publicized.dll</HintPath>
+      <HintPath>G:\C# Applications\SubNauticaMods\DLL\Assembly-CSharp-firstpass_publicized.dll</HintPath>
     </Reference>
     <Reference Include="Assembly-CSharp_publicized">
-      <HintPath>..\DependenciesUnity5\Assembly-CSharp_publicized.dll</HintPath>
+      <HintPath>G:\C# Applications\SubNauticaMods\DLL\Assembly-CSharp_publicized.dll</HintPath>
```
Comment (PrimeSonic) [inline CyclopsBioReactor/CyclopsBioReactor.csproj]: Please undo the changes to the file paths for these.

File: MoreCyclopsUpgrades/MoreCyclopsUpgrades.csproj [hunk 2]
```diff
@@ -45,11 +45,11 @@
     </Reference>
     <Reference Include="Assembly-CSharp-firstpass_publicized, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\DependenciesUnity5\Assembly-CSharp-firstpass_publicized.dll</HintPath>
+      <HintPath>G:\C# Applications\SubNauticaMods\DLL\Assembly-CSharp-firstpass_publicized.dll</HintPath>
     </Reference>
     <Reference Include="Assembly-CSharp_publicized, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\DependenciesUnity5\Assembly-CSharp_publicized.dll</HintPath>
+      <HintPath>G:\C# Applications\SubNauticaMods\DLL\Assembly-CSharp_publicized.dll</HintPath>
```
Comment (PrimeSonic) [inline MoreCyclopsUpgrades/MoreCyclopsUpgrades.csproj]: Please revert changes to this csproj file as well

File: CyclopsBioReactor/Items/CyBioReactor.cs [hunk 3]
```diff
@@ -95,10 +113,7 @@ public override GameObject GetGameObject()
             constructible.techType = this.TechType;
             constructible.model = model;
 
-            // Set the custom texture
-            Texture2D customTexture = ImageUtils.LoadTextureFromFile(@"./QMods/CyclopsBioReactor/Assets/CyBioReactorT.png");
```
Comment (PrimeSonic) [inline CyclopsBioReactor/Items/CyBioReactor.cs]: If you haven't already, you can delete the `CyBioReactorT.png` file as part of this branch.  
It's no longer needed.

**Comment by ccgould** [comment]
Almost there
